samples/net/echo_server/src/common.h

```diff
@@ -16,7 +16,8 @@ void stop_tcp(void);
 
```

Comment:
Typos in commit message of "samples: net: echo_server: Fix handling of large IPv6 header"

s/single buffer net_buf_pull/single buffer then net_buf_pull/
s/and pull of the header/and pull the header/

The last sentence needs clarification, perhaps something like "After this, the packet payload can be copied correctly when replying messages."